contrib/analysis/generate-pbench-timeseries-graphs

```diff
@@ -58,6 +58,17 @@ parser.add_argument('-n', "--dont-generate-graphs", action='store_true', help="D
 parser.add_argument('files', metavar='file', type=str, nargs="*", help="files")
 args = parser.parse_args()
 
+match_res = []
+nomatch_re = None
+initialized_matches = False
```

Comment:
Is this variable ever used anywhere else?

Reply:
Not that I could find....  It seems pointless, now.

Reply:
Yeah, I suppose not any more.  It was intended for lazy evaluation, but if I'm going to statically evaluate these lists, no need for it at all.